### PR TITLE
Fixed linter and flow error as part of the prerelease

### PR DESCRIPTION
**Pull request by stefaniacardenas**
# Task
- Fix linter and flow errors
- Bump next release version

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [ ] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [ ] Have new automated tests been implemented?
- [ ] Have new manual tests been written down?
- [ ] Have tests passed locally?
- [ ] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-394-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-394-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://394-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/Liveness/Intro.js [hunk 1]
```diff
@@ -14,31 +14,36 @@ type Props = {
...
-      <button
-        className={classNames(theme.btn, theme['btn-primary'], theme['btn-centered'])}
-        onClick={preventDefaultOnClick(nextStep)}>
-        {i18n.t('capture.liveness.intro.continue')}
-      </button>
-    </div>
-  </div>
-)
+class Intro extends React.Component<Props> {
+  render() {
```
Comment (rfreitas) [inline src/components/Liveness/Intro.js]: why change a pure component to a non pure component if there is no state?

File: src/components/Liveness/index.js [hunk 2]
```diff
@@ -22,7 +22,7 @@ const initialState = {
   hasError: false,
 };
 
-export default class Liveness extends Component<CameraType, State> {
+export default class Liveness extends React.Component<CameraType, State> {
```
Comment (rfreitas) [inline src/components/Liveness/index.js]: why are we not using preact directly here?

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-394-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-394-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
so do you know how to make the linter fail the `npm check` ?

**Comment by rfreitas** [comment]
Travis automatic deployment: https://394-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-394-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-394-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://394-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-394-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-394-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://394-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by stefaniacardenas** [comment]
@rfreitas I think eslint errors are failing silently but I will need to do some investigation to find that out. I'll open a new ticket for that.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-394-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-394-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://394-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**